grakn-core/src/main/java/ai/grakn/util/ErrorMessage.java

```diff
@@ -146,24 +146,13 @@
     NO_LABEL_SPECIFIED_FOR_HAS("no label was specified for a resource type in a 'has' property"),
     MULTIPLE_GRAPH("a graph has been specified twice for this query"),
 
-    INSERT_RULE_WITHOUT_WHEN("the rule '%s' doesn't have a when specified"),
```

Comment:
I agree!

These particular messages haven't actually been moved though, they've all been deleted and replaces with more general messages.